File: Question6/Question6.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text.RegularExpressions;

public class Solution
{
    public static string solution(int n, List<int> flows)
    {
        // Write solution here
        return "";
    }

    public static void Main()
    {
        int n = Convert.ToInt32(Console.ReadLine());

        var flows = new List<int>();
        string input = Console.ReadLine();
        string pattern = " ";
        String[] substrings = Regex.Split(input, pattern);
        foreach (string s in substrings)
        {
            flows.Add(Convert.ToInt32(s));
        }

        string answer = solution(n, flows);
        Console.WriteLine(answer);
    }
}

File: Question6/Question6.cpp
#include <iostream>
#include <vector>

using namespace std;

void solution(int n, vector<int> flows)
{
    // Write solution here
}

int main()
{
    int n;
    cin >> n;

    vector<int> flows;
    for (int i = 0; i < n; i++)
    {
        int flow;
        cin >> flow;
        flows.push_back(flow);
    }

    solution(n, flows);

    return 0;
}
